source/dofs/dof_handler.cc

```diff
@@ -2648,6 +2658,23 @@ DoFHandler<dim, spacedim>::distribute_mg_dofs()
   internal::DoFHandlerImplementation::Implementation::reserve_space_mg(*this);
   this->mg_number_cache = this->policy->distribute_mg_dofs();
 
+  // cache locally-relevant degrees of freedom on the levels in the form of a
+  // partitioner
+  for (unsigned int level = 0;
+       level <= this->get_triangulation().n_global_levels();
```

Comment:
This is the bug the CI complains about
```suggestion
       level < this->get_triangulation().n_global_levels();
```